app/tests/integration/nimbus/test_desktop_client_integrations.py

```diff
@@ -190,3 +199,98 @@ def test_check_telemetry_enrollment_unenrollment(
     else:
         if control is not False:
             assert False, "Experiment unenrollment was never seen in Ping Data"
+
+
+@pytest.mark.nimbus_integration
+@pytest.mark.xdist_group(name="group2")
+def test_check_telemetry_pref_flip(
+    base_url,
+    selenium,
+    kinto_client,
+    slugify,
+    experiment_name,
+    create_desktop_experiment,
+    experiment_default_data,
+    check_ping_for_experiment,
+    telemetry_event_check,
+):
+    _row_locator = (By.CSS_SELECTOR, "tr > td > span > span")
+    _search_bar_locator = (By.ID, "about-config-search")
+
+    requests.delete("http://ping-server:5000/pings")
+    targeting = helpers.load_targeting_configs()[0]
+    experiment_slug = str(slugify(experiment_name))
+    experiment_default_data["targetingConfigSlug"] = targeting
+    experiment_default_data["featureConfigId"] = 9
+    experiment_default_data["referenceBranch"] = {
```

Comment:
Oh looks like you removed the `treatment_branches` key altogether, but then the default includes a treatment branch here:

https://github.com/mozilla/experimenter/blob/771038bf387170f3e3aceeeaccb7e8f58dff7c27/app/tests/integration/nimbus/conftest.py#L450-L458

Don't you need to explicitly set it to `[]`?

Reply:
If the feature is disabled do I need to remove it? Looks like my hunch was right, it wont create the experiment without a treatment branch being sent to the GQL api call

Reply:
Weird, so locally this failed but on CI it passed